index.md

```diff
@@ -4,19 +4,18 @@ layout: page
 
 # Free PDK Documentation
 
-*Free PDK* is an effort to create an open source alternative to the proprietary
-[Padauk](http://www.padauk.com.tw/index_en.aspx) µC programmer,
-as well as adding support to [SDCC](http://sdcc.sourceforge.net/) for Padauk µCs.
+*Free PDK* is an effort to create an open source toolchain for 8-Bit Microcontrollers by the Taiwanese company
```

Comment:
This almost reads as though Padauk is the one behind this open source toolchain.

Wonder if something like this would be better?

"Free PDK is an effort to create an open source toolchain for the Padauk 8-bit Microcontrollers, as an alternative to the proprietary and closed source tools provided by the Taiwanese company."

Reply:
see change proposal. should be less confusing now.

Reply:
better than before, but still reads a bit ackward to me.

How about?

*Free PDK* is an open sourced and independently created tool-chain for the [Padauk](http://www.padauk.com.tw/index_en.aspx) 8-Bit Microcontrollers,
created as an alternative to the proprietary and closed tools provided by the Taiwanese manufacturer.

Reply:
Sounds good.